policy/release/rpm_packages/rpm_packages.rego

```diff
@@ -0,0 +1,67 @@
+#
+# METADATA
+# title: RPM Packages
+# description: >-
+#   Rules used to verify different properties of specific RPM packages found in the SBOM of the
+#   image being validated.
+#
+package policy.release.rpm_packages
```

Comment:
Should have been just rpm_packages IIUC, see https://github.com/enterprise-contract/ec-policies/pull/1379